Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14.18.0-stretch-slim
+LABEL maintainer="Andrey Ivanov (https://github.com/343dev)"
+
+WORKDIR /app
+
+COPY . .
+
+ENV NODE_ENV="production"
+
+RUN apt update \
+  && apt install --yes --no-install-recommends build-essential libjpeg-dev libpng-dev pkg-config \
```

Comment:
**libpng-dev** — is for guetzli.
**libjpeg-dev** — is for jpegoptim.

Reply:
Just curious. Can we replace -dev-deps with non-dev?

I've found this: https://packages.debian.org/ru/sid/libpng-dev. It looks like the package contains `libpng16-16` that is used for the optimization and other files that not. Can we use `libpng16-16` instead of `libpng-dev`?

Not sure about `libjpeg-dev` though. But https://packages.debian.org/ru/sid/libjpeg-dev contains https://packages.debian.org/ru/sid/libjpeg62-turbo-dev which contains https://packages.debian.org/ru/sid/libjpeg62-turbo, which looks like the lib that does the thing.

Reply:
Good catch. This thing will save another 21 MB.

Reply:
Image total: 221 MB 😇